### PR TITLE
Migrate to Py3

### PR DESCRIPTION
**Pull request by devppjr**
https://github.com/thumbor-community/redis/issues/6

### CODE REVIEW

**Comment by devppjr** [comment]
@masom @phoet ping

**Comment by phoet** [comment]
@devppjr hey sorry, completely out of using python or thumbor. @masom you still around to handle stuff?

**Review by Bladrak — APPROVED** [review]

**Comment by Bladrak** [comment]
You can probably remove the `circleci.yml` file as well, and did you check #7 for the workflows? Might be worth integrating the changes as well :-)

File: setup.py [hunk 1]
```diff
@@ -3,15 +3,31 @@
...
+
+RUNTIME_REQUIREMENTS = [
+    "redis==4.*,>=4.1.3",
+    "thumbor==7.*,>=7.0.6",
+    "pre-commit==2.*,>=2.17.0",
+]
+
 setup(
     name="tc_redis",
     version="1.0.1",
```
Comment (masom) [inline setup.py:33]: The version probably needs to be bumped given this would be a breaking change? Something like `1.1` ?

Comment (RaphaelVRossi) [reply setup.py:33]: We did a lot of breaking changes, migrating from python 2 to python 3. So, IMHO something like to `2.0.0` sounds better.

**Comment by RaphaelVRossi** [comment]
> You can probably remove the `circleci.yml` file as well, and did you check #7 for the workflows? Might be worth integrating the changes as well :-)

We removed `circle.yml`, and the others modifications from #7, we worked with @marcelometal, so you can probably close it.

**Review by Bladrak — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (devppjr) [reply setup.py:33]: @masom ping

Comment (masom) [reply setup.py:33]: @devppjr The version needs to be bumped, and it seems there's an agreement on `2.0.0` given the Python 3 requirement. Bump it :)

Comment (masom) [reply setup.py:33]: @devppjr I've added you as a maintainer of this repo, thanks for this substantial contribution.

**Comment by devppjr** [comment]
I bumped the version (https://github.com/thumbor-community/redis/commit/e4dea465e1f388173083143dbc0942caa143ef48), can you guys publish this in pypi repository? @masom @Bladrak

**Comment by masom** [comment]
Yup, something I'll have to check is how to allow publishing to pypi without my involvement.

**Comment by Bladrak** [comment]
@masom feel free to look at how it's done in tc_aws :)

**Comment by fjallot** [comment]
@devppjr thanks for the work on this PR, I'm working on making thumbor7 work in a 3.9 python docker image on ARM and definitely needed this.
@masom do you think you could publish 2.0.0 to pypi soon ? Doesn't seem to be available yet.
Thanks in advance guys.

**Comment by masom** [comment]
@fjallot working on it right now.

**Comment by masom** [comment]
@devppjr if you could post your pypi username I can (or @Bladrak) add you as a maintainer there as well.

**Comment by masom** [comment]
`2.0.0` is live!

https://pypi.org/project/tc-redis/2.0.0/

**Comment by devppjr** [comment]
> @devppjr if you could post your pypi username I can (or @Bladrak) add you as a maintainer there as well.

Thanks @masom. My pypi username is **devppjr**. Can u add the **metal** and **raphaelrossi** users as a maintainer too? (of this repo too)

**Pull request merged**